[2016-02-17 14:57:44.816] [INFO] log_file - info test ========
[2016-02-17 14:57:44.819] [WARN] log_file - warn
[2016-02-17 14:57:44.819] [ERROR] log_file - error
[2016-02-17 15:00:53.837] [INFO] log_file - info test ========
[2016-02-17 15:00:53.839] [WARN] log_file - warn
[2016-02-17 15:00:53.839] [ERROR] log_file - error
[2016-02-17 15:01:57.287] [INFO] log_file - info test ========
[2016-02-17 15:01:57.289] [WARN] log_file - warn
[2016-02-17 15:01:57.290] [ERROR] log_file - error
[2016-02-17 15:02:22.561] [INFO] log_file - info test ========
[2016-02-17 15:02:22.563] [WARN] log_file - warn
[2016-02-17 15:02:22.564] [ERROR] log_file - error
[2016-02-17 15:02:28.822] [INFO] log_file - info test ========
[2016-02-17 15:02:28.825] [WARN] log_file - warn
[2016-02-17 15:02:28.825] [ERROR] log_file - error
[2016-02-17 15:04:19.275] [INFO] log_file - info test ========
[2016-02-17 15:04:19.277] [WARN] log_file - warn
[2016-02-17 15:04:19.277] [ERROR] log_file - error
[2016-02-18 17:22:31.987] [ERROR] log_file - Error occured at RedisClient.js [redisCli.getValue]
[2016-02-18 17:23:01.086] [ERROR] log_file - Error occured at RedisClient.js [redisCli.getValue]Error: GET can't be processed. The connection has already been closed.
[2016-02-18 17:52:30.445] [ERROR] log_file - { [Error: GET can't be processed. The connection has already been closed.] command: 'GET' }
Error: GET can't be processed. The connection has already been closed.
    at RedisClient.send_command (e:\WebSite\ModuleLearn\node_modules\redis\index.js:788:23)
    at RedisClient.(anonymous function).RedisClient.(anonymous function) (e:\WebSite\ModuleLearn\node_modules\redis\index.js:966:25)
    at getValue (e:\WebSite\ModuleLearn\RedisClient.js:55:24)
    at Object.exports.initTest (e:\WebSite\ModuleLearn\UnitTest\RedisTest.js:12:14)
    at Socket.<anonymous> (e:\WebSite\ModuleLearn\app.js:49:18)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at Pipe.onread (net.js:523:20)
[2016-02-23 16:02:30.820] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:02:48.451] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:02:48.456] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:02:48.461] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:07:57.265] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:07:57.270] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:376:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:07:57.278] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:376:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:08:20.124] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:08:20.131] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:08:20.136] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:08:55.928] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:08:55.934] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:376:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:08:55.940] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:376:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:14:16.699] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:14:16.703] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:376:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:14:16.704] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:376:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:14:48.429] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:14:48.436] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:14:48.441] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:20:56.764] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:20:56.771] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:20:56.779] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:22:37.326] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:22:37.332] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:22:37.336] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:34:23.879] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:34:23.886] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:34:23.893] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:42:01.249] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn/../doc',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:42:01.255] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:42:01.259] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:42:39.157] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn\\..\\doc',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:42:39.161] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:42:39.169] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:44:00.921] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:44:00.927] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:44:00.932] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:44:18.122] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn\\..\\doc',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:44:18.127] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:44:18.132] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:45:56.777] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn\\..\\doc',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:48:39.924] [INFO] log_file - { cwd: 'e:\\WebSite\\ModuleLearn\\..\\doc',
  username: 'ericjiang',
  password: 'Shuanglong3' }
[2016-02-23 16:48:39.931] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 16:48:39.938] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 17:13:13.610] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 17:13:13.619] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 17:14:14.566] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-23 17:14:14.573] [ERROR] log_file - [Error: [SVN ERROR:404] svn command not found]
Error: [SVN ERROR:404] svn command not found
    at ChildProcess.<anonymous> (e:\WebSite\ModuleLearn\node_modules\node.svn\svn.js:375:15)
    at emitOne (events.js:77:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:441:9)
    at process._tickCallback (node.js:355:17)
[2016-02-24 14:10:47.071] [INFO] log_file - isBuffer
[2016-02-24 14:16:47.723] [INFO] log_file - is buffer
[2016-02-24 14:21:08.846] [INFO] log_file - is buffer: 
[2016-02-24 14:21:35.914] [INFO] log_file - is buffer: 
[2016-02-24 14:21:53.853] [INFO] log_file - is buffer: true
[2016-02-24 14:23:56.486] [INFO] log_file - Path: turn.xml
Name: turn.xml
Working Copy Root Path: E:\WebSite\client
URL: http://svn.taomee.com/seer2/client/trunk/Client/turn.xml
Repository Root: http://svn.taomee.com/seer2/client
Repository UUID: de0dd7c9-4e81-4b5e-b3e8-f71bddc60c3f
Revision: 104840
Node Kind: file
Schedule: normal
Last Changed Author: andre
Last Changed Rev: 67562
Last Changed Date: 2014-04-11 01:50:00 +0800 (周五, 11 四月 2014)
Text Last Updated: 2016-02-24 11:16:46 +0800 (周三, 24 二月 2016)
Checksum: 3a7346ed73429cfeff4e1f862ac4bbb9a4a46617


[2016-02-24 14:24:41.316] [INFO] log_file - Path: turn.xml
Name: turn.xml
Working Copy Root Path: E:\WebSite\client
URL: http://svn.taomee.com/seer2/client/trunk/Client/turn.xml
Repository Root: http://svn.taomee.com/seer2/client
Repository UUID: de0dd7c9-4e81-4b5e-b3e8-f71bddc60c3f
Revision: 104840
Node Kind: file
Schedule: normal
Last Changed Author: andre
Last Changed Rev: 67562
Last Changed Date: 2014-04-11 01:50:00 +0800 (周五, 11 四月 2014)
Text Last Updated: 2016-02-24 11:16:46 +0800 (周三, 24 二月 2016)
Checksum: 3a7346ed73429cfeff4e1f862ac4bbb9a4a46617


[2016-02-24 14:25:34.132] [INFO] log_file - Path: turn.xml
Name: turn.xml
Working Copy Root Path: E:\WebSite\client
URL: http://svn.taomee.com/seer2/client/trunk/Client/turn.xml
Repository Root: http://svn.taomee.com/seer2/client
Repository UUID: de0dd7c9-4e81-4b5e-b3e8-f71bddc60c3f
Revision: 104840
Node Kind: file
Schedule: normal
Last Changed Author: andre
Last Changed Rev: 67562
Last Changed Date: 2014-04-11 01:50:00 +0800 (周五, 11 四月 2014)
Text Last Updated: 2016-02-24 11:16:46 +0800 (周三, 24 二月 2016)
Checksum: 3a7346ed73429cfeff4e1f862ac4bbb9a4a46617


[2016-02-24 14:25:44.825] [INFO] log_file - Path: turn.xml
Name: turn.xml
Working Copy Root Path: E:\WebSite\client
URL: http://svn.taomee.com/seer2/client/trunk/Client/turn.xml
Repository Root: http://svn.taomee.com/seer2/client
Repository UUID: de0dd7c9-4e81-4b5e-b3e8-f71bddc60c3f
Revision: 104840
Node Kind: file
Schedule: normal
Last Changed Author: andre
Last Changed Rev: 67562
Last Changed Date: 2014-04-11 01:50:00 +0800 (周五, 11 四月 2014)
Text Last Updated: 2016-02-24 11:16:46 +0800 (周三, 24 二月 2016)
Checksum: 3a7346ed73429cfeff4e1f862ac4bbb9a4a46617


